sdk/translation/ai-translation-text/test/public/dictionaryLookupTest.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Instead of duplicating the tests, could we make them parameterized by the client?

Reply:
I see this as not addressed yet so I will mark it as unresolved. Do you think it can be addressed?

Reply:
I added message, that we will re-visit this for GA. But my comments were deleted.